### ISSUE TITLE
`<onlyfor>` component tag for markdown reader

### ISSUE THREAD

**Issue by alesanchezr** [4GeeksAcademy/About-4Geeks-Academy#3182]
The markdown read must interpret a new tag `<onlyfor>`.

When found, the text included in the innerhtml of the component will only be shown for some people based on the `permissions` that they have. For example:
```
<onlyfor permission="read_private_lesson">
hello world
<onlyfor>
```

In the block of content above, if the user does not have `read_private_lesson` permission it will not be able to read `hellow world`

Instead, we will display a box and message saying: "We are sorry, you don't have enough privileges to access this block of content, please signup or upgrade your plan to access it."

If the user is not logged in at all, the box will say: "Want to read more? Register or login with your email to continue reading." and it will display the a form with the email box to start the signup or login process. Once the emails is inputed and the `continue` button is pressed, it will be taken to the signup form with the email already pre-filled.

Here is an example:

![main-qimg-1cab832f6a14ef10306a97db34274061](https://user-images.githubusercontent.com/426452/188219813-a2beb1bf-d546-4f03-b058-66100679bf3b.png)

**Comment by alesanchezr** [comment]
Here is [the design in figma](https://www.figma.com/file/hJsE2D6fu3peR49mt6kZD1/4Geeks.com?node-id=4350%3A11695) and an example:

![Image](https://user-images.githubusercontent.com/426452/191370054-5badf274-94b0-4b66-a14a-dbd67f68eec1.png)

**Comment by ArielCalisaya** [comment]
>Issue solved here in [PR #423](https://github.com/breatheco-de/app/pull/423)

**Comment by alesanchezr** [comment]
Waiting for backend user permission list on the /me request.
